src/components/WwwFrame/TheHeader.vue

```diff
@@ -112,33 +112,34 @@
 						</div>
 
 						<template v-if="lendMenuButtonExp">
```

Comment:
Is this experiment disabled?

Reply:
Yes it's currently disabled, but we'll enable it for this new variation.

Reply:
It's not enabled on dev.